cmd/skupper/skupper.go

```diff
@@ -156,10 +157,22 @@ func silenceCobra(cmd *cobra.Command) {
 }
 
 func NewClient(namespace string, context string, kubeConfigPath string) *client.VanClient {
+	return NewClientHandleError(namespace, context, kubeConfigPath, true)
+}
+
+func NewClientHandleError(namespace string, context string, kubeConfigPath string, exitOnError bool) *client.VanClient {
```

Comment:
(You could further simplify, by removing the exitOnError flag, returning the error and handling that error by exiting in NewClient(). The newClientSansExit() function could then just ignore the error, or set some variable if there is ever a future need for it).